csharp/src/ProtocolBuffers/CodedInputStream.cs

```diff
@@ -793,12 +608,12 @@ private uint SlowReadRawVarint32()
         }
 
         /// <summary>
-        /// Read a raw Varint from the stream.  If larger than 32 bits, discard the upper bits.
+        /// Reads a raw Varint from the stream.  If larger than 32 bits, discard the upper bits.
         /// This method is optimised for the case where we've got lots of data in the buffer.
         /// That means we can check the size just once, then just read directly from the buffer
         /// without constant rechecking of the buffer length.
         /// </summary>
-        public uint ReadRawVarint32()
+        internal uint ReadRawVarint32()

```

Comment:
Will external code still have a way to use the Raw methods? They are probably not terribly useful but having access to the low level encoding routines might come handy from time to time if someone tries to hack something non-standard (or perhaps in tests).